test_employee_review.rb

```diff
@@ -0,0 +1,83 @@
+require "minitest/autorun"
+require 'minitest/pride'
+require './department.rb'
+require './employee.rb'
+
+class EmployeeReviewTest < Minitest::Test
+
+  def test_01_department_class_takes_arguments
+    assert Department.new("Accounting")
+  end
+
+  def test_02_employee_class_takes_two_arguments
+    assert Employee.new("Chad Wellington", "72,000")
```

Comment:
OH, I see why you did to_i now.  Rather than passing in strings for salaries, though, it would better to pass in numbers, like this:

assert Employee.new("Chad Wellington", 72000)

That way, you won't need to do to_i inside your methods, and you won't have to have tests like this later:

department_1.total_salary == "67000"

Just keep them all in numbers, and things will be simpler for you.